torch/backends/cudnn/rnn.py

```diff
@@ -252,7 +254,15 @@ def forward(fn, input, hx, weight, output, hy):
         # Alternatively, _copyParams could be written more carefully.
         w.zero_()
         params = get_parameters(fn, handle, w)
+        if fn.skip_input:
+            for layer_index in range(fn.num_directions):
+                params[layer_index][0] = None
+                params[layer_index][2] = None
```

Comment:
Can you add a comment saying why 0 and 2 (sth like `params[layer_index] == [wi, bi, wh, bh]`)? Does it work for all kinds of cells?